src/topology/algebra/mul_action.lean

```diff
@@ -58,6 +58,11 @@ lemma filter.tendsto.const_smul {f : β → α} {l : filter β} {a : α} (hf : t
   tendsto (λ x, c • f x) l (𝓝 (c • a)) :=
 tendsto_const_nhds.smul hf
 
+lemma filter.tendsto.smul_const {f : β → M} {l : filter β} {c : M}
+  (hf : filter.tendsto f l (nhds c)) (a : α) :
+  filter.tendsto (λ x, (f x) • a) l (nhds (c • a)) :=
```

Comment:
You should be able to delete these `filter.`s.
```suggestion
  (hf : tendsto f l (nhds c)) (a : α) :
  tendsto (λ x, (f x) • a) l (nhds (c • a)) :=
```

Reply:
Oh oops, of course!